runtime/runtime.go

```diff
@@ -58,8 +58,19 @@ type Configuration struct {
 	StateMountLocation string
 	// StateMasterNamespace the namespace where state created by bundles will be copied to between actions
 	StateMasterNamespace string
+	// Platform indicates which platform the runtime is for
+	Platform platform
```

Comment:
I will remove as probably shouldn't be part of this PR. The reason it came about was I noticed when trying to update the tests that I needed to initialize the runtime provider. However this now tries to call out to OpenShift to figure out what type of cluster / platform it is running in. I added this as I thought the caller of bundle should be in a position to know / specify where they are running rather than the lib deciding?